pkg/management/authorizationutil.go

```diff
@@ -53,3 +54,42 @@ func checkTarget(authorizations []configuration.Authorization, allowedTargetReal
 	}
 	return nil
 }
+
+// Validate the scope of each authorization in the array. Returns an error if an authorization is not valid
+func ValidateScope(authorizations []configuration.Authorization) error {
+	for _, authz := range authorizations {
+		scope, err := getScope(authz)
+		if err != nil {
+			return err
+		}
+
+		if authz.TargetRealmID == nil {
+			return errors.New("missing target realm")
```

Comment:
We should return more specific errors in this validation function:
`return errorhandler.CreateBadRequestError(constants.MsgErrMissingParam + "." + constants.Authorization+".targetRealm")` ?

Reply:
I did that to have the same kind of error as the method Validate from the same file, I then returned `errorhandler.CreateBadRequestError(constants.MsgErrInvalidParam + "." + constants.Authorization)` when I used it. Example : https://github.com/cloudtrust/keycloak-bridge/blob/4961ed1d33150cddad65dbaaa7db1b6eddc37eff/pkg/management/component.go#L1475